project/gh-action/docker/app.py

```
@@ -0,0 +1,22 @@
+from flask import Flask, request
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)